File: Exercies/07_while/e06_evenNumberSum.py
lower = int(input("Zadajte dolný limit: "))
upper = int(input("Zadajte horný limit: "))
total = 0
for i in range(lower, upper + 1):
    if i % 2 == 0:
        total += i
print(total)

lower = int(input("Zadajte dolný limit: "))
upper = int(input("Zadajte horný limit: "))
total = 0
i = lower
while i <= upper:
    if i % 2 == 0:
        total += i
    i += 1
print(total)

File: Answers/07_while/e06_evenNumberSum.cs
using System;

class Program
{
    static void Main()
    {
        int lower = int.Parse(Console.ReadLine());
        int upper = int.Parse(Console.ReadLine());
        int total = 0;

        for (int i = lower; i <= upper; i++)
        {
            if (i % 2 == 0)
            {
                total += i;
            }
        }
        Console.WriteLine(total);

        lower = int.Parse(Console.ReadLine());
        upper = int.Parse(Console.ReadLine());
        total = 0;
        int j = lower;

        while (j <= upper)
        {
            if (j % 2 == 0)
            {
                total += j;
            }
            j++;
        }
        Console.WriteLine(total);
    }
}
